chandra_aca/star_probs.py

```diff
@@ -66,7 +66,7 @@ def set_acq_model_ms_filter(ms_enabled=False):
     MULT_STARS_ENABLED = ms_enabled
 
 
-def t_ccd_warm_limit(mags, date=None, colors=0, min_n_acq=5.0,
+def t_ccd_warm_limit(mags, date=None, colors=0, halfwidths=120, min_n_acq=5.0,
```

Comment:
I wasn't sure if it made the most sense to have a default halfwidth or not, but colors had a default so I went with that.

Reply:
I don't see any github code using t_ccd_warm_limit with fixed argument order (so I felt free to stick halfwidth after colors).  There is some ambiguity in the star_probs code for when these things are plural (halfwidth vs halfwidths).

Reply:
The intent with naming was that params that *can* be a scalar are named with singular while params that only make sense as a list are plural.  So this function needs plural.

Now whether that was a good idea or just confusing (and whether it is applied uniformly) is a completely different matter.

Reply:
Right, I figured that was the intent, but yes,  I'm just feeding these into `acq_success_prob` which is has all its params labeled in the singular and then broadcasts and renames plural.  So a little confusing.